ironfish/src/primitives/rawTransaction.test.slow.ts

```diff
@@ -46,7 +46,7 @@ function createTestRawTransaction(
     new NativeNote(
       account.publicAddress,
       123456789n,
-      'some memo',
+      Buffer.from('some memo'),
```

Comment:
This is sort of misleading right? Shouldn’t the buffer be from a hex encoded string?

Reply:
Never mind I misread this, it will be a satisfactory buffer.